packages/browser/__tests__/login/oidc/redirectHandler/AuthCodeRedirectHandler.spec.ts

```diff
@@ -38,6 +38,35 @@ import {
 } from "../../../../src/login/oidc/redirectHandler/AuthCodeRedirectHandler";
 import { RedirectorMock } from "../../../../src/login/oidc/__mocks__/Redirector";
 import { SessionInfoManagerMock } from "../../../../src/sessionInfo/__mocks__/SessionInfoManager";
+import { KEY_CURRENT_ISSUER } from "../../../../dist/constant";
+
+// Not sure this is strictly necessary, but perhaps best to mock out
+// 'localStorage' in case of parallelized tests.
```

Comment:
I think it's safe to remove this comment, mocking out localStorage does make sense.

Reply:
Sure - 9d54085